index.html

```diff
@@ -1,277 +1,268 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+
+  <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Alex Portfolio</title>
-</head>
-<body>
+  </head>
+
+  <body>
     <!--Principal nav-->
     <header>
-        <nav>
-          <a href="">Alexander</a>
-          <ul>
-            <li class="nav-list menu-collapsed" ><a href="">
-                <div class="menu-line"></div>
-                <div class="menu-line"></div>
-                <div class="menu-line"></div>
-            </a></li>
-          </ul>
-          
-          <ul class="navbar-items">
-            <li class="nav-list"><a href="">Portfolio</a></li>
-            <li class="nav-list"><a href="">About</a></li>
-            <li class="nav-list"><a href="">Contact</a></li>
-           
-          </ul>
-        </nav>
+      <nav class="nav-style">
+        <a href="#" class="text-logo" id="logo">Alexander</a>
+
+        <button type="button" id="menu-container" onclick="togglerMenu()">
+          <span class="open-menu menu-collapsed" id="menu-button"></span>
+        </button>
+
+        <ul class="mobile-nav-elements show-menu" id="navigation-elements">
+          <li class="mobile-nav-list active"><a href="#work">Portfolio</a></li>
+          <li class="mobile-nav-list active"><a href="#about">About</a></li>
+          <li class="mobile-nav-list active"><a href="#contact">Contact</a></li>
```

Comment:
- [ ] The divider lines should be in gray as in the Figma design so please it would be better if you stick to the Figma design 

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/167513508-b32dbaa5-4f3e-4944-9cc5-e7368a7c9a50.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/167513531-48ee1735-2c85-43c1-a1ab-7bb8815b6af4.png)